GameData/ROEngines/PartConfigs/RL10A4-2N_BDB.cfg

```diff
@@ -0,0 +1,213 @@
+PART
+{
+	module = Part
+	name = ROE-RL10A4-2N
+	author = CobaltWolf
+
+	category = Engine
+	subcategory = 0
+	RSSROConfig = true
+	RP0conf = true
+
+	crashTolerance = 10
+	maxTemp = 673.15
+	skinTemp = 773.15
+	fuelCrossfeed = true
+	breakingForce = 10000
+	breakingTorque = 10000
+
+	//	============================================================================
+	//	Update Below
+	//	============================================================================
+
+	MODEL
+	{
+		model = ROEngines/Assets/BDB/Centaur/bluedog_Centaur_RL10A4-2-N
+	}
+
+	scale = 1.0
+	rescaleFactor = 1.6
+	node_stack_top = 0.0, 0.48962, 0.0, 0.0, 1.0, 0.0, 1
+	node_stack_bottom = 0.0, -0.53707, 0.0, 0.0, -1.0, 0.0, 1
+	// stack, srfAttach, allowStack, allowSrfAttach, allowCollision
+	attachRules = 1,1,1,1,0
+
+	title = RL10A-4 Series Vacuum Engine
+	manufacturer = Pratt & Whitney
+	description = Hydrolox restartable expander-cycle vacuum engine used in multiple upper stages, including Centaur, the Saturn I S-IV, and the Delta Cryogenic Second Stage. It has low thrust, but very high specific impulse and low mass, making it ideal for high energy, beyond-low-Earth-orbit applications like launching satellites to geostationary transfer orbits or to the Moon or other planets. However, like all hydrolox engines, hydrogen boiloff is a serious issue without heat pumps or radiators.
+
+	tags = rl10, rl10a, rl10a-4-2n, 4, rl-10, hydrolox, centaur
+
+	engineType = RL10
+	stagingIcon = LIQUID_ENGINE
+	stageOffset = 1
+	childStageOffset = 1
+
+	TechRequired = hydrolox1992
+	cost = 500
+	entryCost = 26000
+
+	MODULE
+	{
+		name = ModuleEnginesRF
+		thrustVectorTransformName = thrustTransform
+		engineID = RL10A4
+	}
+
+	MODULE
+	{
+		name = ModuleGimbal
+		gimbalTransformName = gimbalTransform
+		gimbalRange = 4
+		useGimbalResponseSpeed = true
+		gimbalResponseSpeed = 16
```

Comment:
Believe these values will be overridden by RO's engine config.  Should only set the module and transform in the original part def.  If the gimbal needs to be overrridden, that happens in a later pass.